site/content/blog/2025-04-11-hipcheck-3.13.0-release.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Hipcheck 3.13.0 Release
+authors:
+  - Julian Lanson
+extra:
+  author_img: "images/authors/julian.jpg"
+---
+
+Hipcheck 3.13.0 is out! This is a lighter release as far as user-facing changes
+go, but internally we have been doing a lot of refactoring and development to
+prepare for exciting new changes that will be ready in upcoming releases.
+
+<!-- more -->
+
+---
+
+## 📦&nbsp;&nbsp;User-facing Changes
+
+If you missed the previous [blog post][python-blog], the Hipcheck team has
+released a Python SDK; Hipcheck users can now easily write their own data and
+analysis plugins for Hipcheck in Python!
```

Comment:
I'd add a header for the new Python SDK, just to make it clearer so people are less likely to skim over and miss it.